src/registers/model_specific.rs

```diff
@@ -445,4 +488,74 @@ mod x86_64 {
             unsafe { msr.write(value.bits()) };
         }
     }
+
+    impl UCet {
+        /// Read the raw IA32_U_CET.
+        #[inline]
+        fn read_raw() -> u64 {
+            unsafe { Self::MSR.read() }
+        }
+
+        /// Write the raw IA32_U_CET.
+        #[inline]
+        fn write_raw(value: u64) {
+            let mut msr = Self::MSR;
+            unsafe {
+                msr.write(value);
+            }
+        }
+
+        /// Read IA32_U_CET.
```

Comment:
```suggestion
        /// Read IA32_U_CET. Returns a tuple of the flags and the address to the legacy code page bitmap.
```

I missed that the other method also needed that comment, my bad.